text/0000-fabric-website.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: default
+title: Fabric_Website
+nav_order: 3
+---
+
+- Feature Name: Fabric Website
+- Start Date: (fill me in with today's  date, 2020-08-14)
+- RFC PR: (leave this empty)
+- Fabric Issue: (leave this empty)
+
+# Summary
+[summary]: #summary
+
+This RFC details the proposal for a new website for using Hyperledger Fabric. This website would serve as a "front door" for the project, and collate the various resources available to help users get started with Fabric.
+
+The website has been deployed to Heroku for development purposes and is available to browse [here](https://warm-reaches-04614.herokuapp.com/).
+
+# Motivation
+[motivation]: #motivation
+
+Presently, there are multiple different places to get started with Hyperledger Fabric. It can be difficult for a new user to know which resource is the best one for them to start with based on what they would like to achieve. The available documentation has excellent content for getting started and explains everything well, but there is also a large amount of additional content on other relevant topics (for example, performance testing) that is also useful to new users.
```

Comment:
question here for performance testing, see I am currently developing/maintaining some project related with fabric performance such as
https://github.com/guoger/tape
and 
https://github.com/SamYuan1990/Probe
so
what's the plan as generic guide for advance knowledges. such as performance testing, HA, etc...
is there any plan for performance testing specific?
is there any plan for link with other fabric labs project as mini fab?

Reply:
**additional content on other relevant topics** is a good start for us to brainstorm, Can this Fabric resources navigation/indexing Website provides more opportunity for light-weight, third-party community projects in Fabric ecosystem?